glean-core/android/src/test/java/mozilla/telemetry/glean/GleanFromJavaTest.java

```diff
@@ -76,4 +78,20 @@ public void testCanSendPing() {
         // submit() can be called with a `reason`.
         Pings.INSTANCE.baseline().submit(Pings.baselineReasonCodes.inactive);
     }
+
+    @Test
+    public void testPassingExplicitBuildInfo() {
+        Configuration config =
+            new Configuration(Configuration.DEFAULT_TELEMETRY_ENDPOINT, "test-channel");
+        BuildInfo buildInfo =
+            new BuildInfo("foo", "c0ffee");
+        Glean.INSTANCE.initialize(appContext, true, config, buildInfo);
+
+        // The testing API doesn't seem to be available from these Java tests, so the best
```

Comment:
nit: can you reference the bug you just filed about the testing API in java, within this comment?

Reply:
Done